backend-services/server.py

```diff
@@ -0,0 +1,44 @@
+#Flask Instance for Off-Top | Flask is a micro-framework
+
+from flask import Flask, jsonify, request
+from flask.templating import render_template
+from Models.UserSession import UserSession
+from flask_restplus import Api
+from datetime import datetime
+from datetime import timedelta
+from flask import session
+from flask_cors import CORS, cross_origin
+import requests
+import json
+from werkzeug.utils import cached_property # had error ImportError: cannot import name 'cached_property' from 'werkzeug' (C:\off-top-python\my-venv\lib\site-packages\werkzeug\__init__.py)
```

Comment:
Is there anyway you can fix this error? I am unable to get python to build server.py on my computer. Possibly look into this link? https://stackoverflow.com/questions/46457179/python3-cannot-import-name-cached-property